lib/mathematics.rb

```diff
@@ -20,7 +20,7 @@ def standard_deviation
       ((value - find_average)**2)
     end
 
-    result = (sum / (476 - 1))
+    result = (sum / (@merchants.merchant_list.count.to_f - 1))
```

Comment:
Great refactor here so we can reuse this piece of code and update it for future purposes, and not just "hard code" a value in for the merchants that we currently have.